docs/Troubleshooting/MySqlSizeIssue.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Maintenance Troubleshooting"
+sidebar_position: 4
+description: >
+  Maintenance Troubleshooting
+---
+
+## How to manage the quickly increasing MySQL disk consumption?
+
+DevLake is designed to collect data by first deleting the existing data and
+then inserting new data. While this approach ensures that the latest data is
+always available, it leads to a rapid increase in MySQL disk consumption.
+This growth is primarily caused by the large size of the binary logs generated
+after each data collection cycle.
+
+### Why dose DevLake choose to delete the existing data and then insert new data?
+
+Because we want to ensure that the latest data is 
+always available. If we don't delete the existing data, some old data which has been deleted 
+from the source system will still be available in the DevLake database.
```

Comment:
I think "source system" should be replaced with "previous step/subtask" sth like that.